### ISSUE TITLE
Incompatibility of jupyter lab 2.0.x and @jupyter-voila/jupyterlab-preview

### ISSUE THREAD

**Issue by AdamJel** [voila-dashboards/voila#553]
Hello, 

jupyterlab-preview is not compatible with jupyter lab 2.0.0 and does not work. 

When trying to `jupyter labextension update --all`,
result is:
```
...
No compatible version found for @jupyter-voila/jupyterlab-preview!
...
```
Build fails then:
```
(voila) C:\Users\jelinek>jupyter lab build
[LabBuildApp] JupyterLab 2.0.0
[LabBuildApp] Building in C:\tools\miniconda3\envs\voila\share\jupyter\lab
[LabBuildApp] Building jupyterlab assets (build:prod:minimize)
[LabBuildApp] WARNING | The extension "@jupyter-voila/jupyterlab-preview" is outdated.

\
```

**Comment by jtpio** [comment]
Thanks @AdamJel for reporting that.

Indeed we haven't got to do the update yet, but it should come soon.

**Comment by AdamJel** [comment]
I am glad to help.

Since there is already a 2.0.1 jupyterlab release - just in case -> Issue applies to this release as well.

**Comment by jtpio** [comment]
Fixed by #555.

A new version has been published to npm.
